.github/workflows/callable_test-automation.yaml

```diff
@@ -137,6 +137,7 @@ env:
   NETWORK_SUBNET: ${{ fromJSON(inputs.networkJSON).networkSubnet }}
   TEST_DIRECTORY: ${{ inputs.testDirectory }}
   INCLUDE_PAYMENT_TESTS: ${{ inputs.includePaymentTests }}
+  TEST_DIRECTORY_PAYMENT: "system_payments_only/indy-node-tests" # keep sovrin token tests untouched and still depending on libsovtoken and libindy
```

Comment:
The value for `TEST_DIRECTORY_PAYMENT` should be coming from an input so it can be overridden like `TEST_DIRECTORY`.

Reply:
We can move both paths into TEST_DIRECTORY by switch it from string to key-value structure but it can broke your initial pipeline.

Reply:
Ok, we'll look into fixing that later.  @pSchlarb is working on reducing the number of required inputs.